src/containers/main/index.js

```diff
@@ -16,6 +16,7 @@ const mapStateToProps = state => {
     return ({
         searchResult: state.main.search.searchResult,
```

Comment:
please change these property name to not include search (result, text) and change showLoader to fetching